code-challenges.js

```diff
@@ -20,42 +20,90 @@ const fruit4 = "kiwi"
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 // Pseudo code:
+//func named boilingPoint 
+// argument to check if boiling point <,> ,=== 212
+// want to return given temps
+// give parameter with console.log
+
 
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
-
+const boilingPoint = (degrees) => {
+ if ( degrees > 212) {
+ return `${temp2} is above boiling point`
+ } else if (degrees === 212) {
+    return `${temp3} is at boiling point`
+ } else if (degrees < 212) {
+    return `${temp1} is below boiling point`
+ }
+
```

Comment:
This code accomplishes the mission. Now let's go back and see how we can make the function reusable for any numbers.